go/pkg/pwdb/entity.go

```diff
@@ -66,6 +67,8 @@ func ForeignKeys() [][3]string {
 		{"Team", "season_id", "season(id)"},
 		{"TeamMember", "team_id", "team(id)"},
 		{"TeamMember", "user_id", "user(id)"},
+		{"TeamInvite", "team_id, team(id)"},
+		{"TeamInvite", "user_id, user(id)"},
```

Comment:
```suggestion
		{"TeamInvite", "team_id", "team(id)"},
		{"TeamInvite", "user_id", "user(id)"},
```

that's the cause of the issue you had during init

Reply:
argg FML sorry bro :p